src/specs/models/ProcessingConfigBodies.v1.yaml

```diff
@@ -0,0 +1,12 @@
+title: Processing config bodies
+description: Valid processing config bodies.
+anyOf:
+  - $ref: ./processing-config-bodies/ProcessingConfigCellSizeDistribution.v1.yaml
+  - $ref: ./processing-config-bodies/ProcessingConfigClassifier.v1.yaml
+  - $ref: ./processing-config-bodies/ProcessingConfigMitochondrialContent.v1.yaml
+  - $ref: ./processing-config-bodies/ProcessingConfigReadAlignment.v1.yaml
+  - $ref: ./processing-config-bodies/ProcessingConfigClassifier.v1.yaml
+  - $ref: ./processing-config-bodies/ProcessingConfigNumGenesVsNumUMIs.v1.yaml
+  - $ref: ./processing-config-bodies/ProcessingConfigDoubletScores.v1.yaml
+  - $ref: ./processing-config-bodies/ProcessingConfigConfigureEmbedding.v1.yaml
```

Comment:
right now for the configure the embedding UI module, we only send GetEmbedding task to the worker when the user changes the embedding settings. How is this `ProcessingConfigConfigureEmbedding.v1.yaml` going to be used now?

Reply:
The only reason this was done is to ease readability from Stoplight Studio. See: 
![image](https://user-images.githubusercontent.com/16019444/107950798-2ebbd400-6f8f-11eb-849d-bec462d5a6a4.png)